modules/op-release-notes-1-9.adoc

```diff
@@ -418,4 +418,14 @@ With this update, you can use the dynamic template variables instead of hardcodi
 
 * The value for `CATALOG_REFRESH_INTERVAL`, a field in the Hub API `ConfigMap` object used in the Tekton Hub CR, is not getting updated with a custom value provided by the user.
 +
-Workaround: None. You can track the issue link:https://issues.redhat.com/browse/SRVKP-2854[SRVKP-2854].
\ No newline at end of file
+Workaround: None. You can track the issue link:https://issues.redhat.com/browse/SRVKP-2854[SRVKP-2854].
+
+[id="release-notes-1-9-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.2
+
+With this update, {pipelines-title} General Availability (GA) 1.9.2 is available on {product-title} 4.11 and later.
+
+[id="fixed-issues-1-9-2_{context}"]
+== Fixed issues
+
+* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of RedHat OpenShift Container Platform. With this update, this misconfiguration issue has been fixed.
```

Comment:
```suggestion
* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of Red Hat OpenShift Container Platform. With this update, this misconfiguration issue has been fixed.
```

Reply:
Ughh, sorry for the typo. I have fixed this and made the change now.

Reply:
Could you please provide your LGTM @concaf as I have fixed your comment.